rocketpy/simulation/flight.py

```diff
@@ -1963,24 +1998,27 @@ def u_dot_parachute(self, t, u, post_processing=False):
         wind_velocity_x = self.env.wind_velocity_x.get_value_opt(z)
         wind_velocity_y = self.env.wind_velocity_y.get_value_opt(z)
 
-        # Get Parachute data
-        cd_s = self.parachute_cd_s
-
         # Get the mass of the rocket
         mp = self.rocket.dry_mass
 
-        # Define constants
-        ka = 1  # Added mass coefficient (depends on parachute's porosity)
-        R = 1.5  # Parachute radius
         # to = 1.2
         # eta = 1
         # Rdot = (6 * R * (1 - eta) / (1.2**6)) * (
         #     (1 - eta) * t**5 + eta * (to**3) * (t**2)
         # )
         # Rdot = 0
 
+        # tf = 8 * nominal diameter / velocity at line stretch
+
         # Calculate added mass
-        ma = ka * rho * (4 / 3) * np.pi * R**3
+        ma = (
+            self.ka
+            * rho
+            * (4 / 3)
+            * np.pi
+            * self.parachute_radius**2
+            * self.parachute_height
+        )
```

Comment:
This is calculating the volume of the whole sphere right? I think it should be divided by two to account for the hemispheroid shape only

Reply:
@ArthurJWH